.devcontainer/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3"
+
+services:
+  web:
+    volumes:
+      - .:/app:delegated
+      - vscode-server:/root/.vscode-server:cached
+      - /var/run/docker.sock:/var/run/docker.sock:cached
+    # Override command to prevent container crashing if webpack build exits
+    command: /bin/bash -c 'while !</dev/tcp/postgres/5432; do echo "Waiting for postgres databaseto start..."; /bin/sleep 1; done; \
+        /bin/sleep 1; /bin/sh /app/utility/start_server.sh;'
```

Comment:
@jofsky This is now the same as the command in the root-level docker-compose.yml. So it's not actually overriding anything and should be removed instead of repeating ourselves.

Reply:
Ok this all makes sense now to me.

Reply:
@jofsky but you didn't fix it?